test/e2e/auth/node_authz.go

```diff
@@ -174,6 +174,12 @@ var _ = SIGDescribe("[Feature:NodeAuthorizer]", func() {
 		}
 		ginkgo.By(fmt.Sprintf("Create node foo by user: %v", asUser))
 		_, err := c.CoreV1().Nodes().Create(context.TODO(), node, metav1.CreateOptions{})
+		defer func() {
+			err := f.ClientSet.CoreV1().Nodes().Delete(context.TODO(), node.Name, metav1.DeleteOptions{})
```

Comment:
It is better to write NOTE why we need this cleanup, because in the expected test path the node creation is failed and we don't need to do cleanup here.

Reply:
Done, wrote a comment explaining it.